src/loki_alert_rules/log_volume.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: high-log-volume
+    rules:
+      - alert: HighLogVolume
+        expr: |
+          count_over_time(({%%juju_topology%%})[30s]) > 100
```

Comment:
In charm code we already replace the stub with "job" (a label we know [we add to everything](https://github.com/canonical/loki-k8s-operator/issues/406)). Should we avoid using the stub to begin with?